GG/src/DropDownList.cpp

```diff
@@ -30,6 +30,7 @@
 #include <GG/Scroll.h>
 #include <GG/StyleFactory.h>
 #include <GG/WndEvent.h>
+#include "../util/Logger.h"
```

Comment:
we've avoided putting any FreeOron-specific code / headers into GG code, historically because it was / is supposed to be an independent library that happens to only be used by FreeOrion...